### PR TITLE
Add NOAA and Brasil mirrors

### PR DESCRIPTION
**Pull request by PaulWessel**
The last two mirrors for now - still awaiting UNAVCO.

### CODE REVIEW

File: mirrors/index.rst [hunk 1]
```diff
@@ -51,6 +51,10 @@ We thank the hosting institutions for mirroring the GMT data distribution so tha
      - https://oceania.generic-mapping-tools.org
      - SOEST, U of Hawaii, USA
      - .. image:: https://img.shields.io/website?down_message=offline&label=%20&style=plastic&up_message=OK&url=https%3A%2F%2Foceania.generic-mapping-tools.org/gmt_data_server.txt
+   * - **Brasil**
```
Comment (seisman) [inline mirrors/index.rst:54]: Brasil or Brazil?

**Comment by PaulWessel** [comment]
Well, I googled that actually.  The country calls itself Brasil.  That some countries call it Brazil is not really a good reason to use that I think, especially since most users of that mirror are likely to be from Brasil or S America in general.

**Comment by PaulWessel** [comment]
Perhaps a Portuguese like @joa-quim will have opinions here?

**Comment by joa-quim** [comment]
It's ``Brasil`` but I'm not sure ``Brazil`` is completely wrong. It's the ancient (quite old I think) form.

**Comment by PaulWessel** [comment]
I could add another forward called brazil but undocumented.  I am planning to add Portugal anyway and leave Europe under the hood.

**Pull request merged**